openproblems/tasks/denoising/methods/dca.py

```diff
@@ -0,0 +1,29 @@
+from ....tools.decorators import method
+from ....tools.utils import check_version
+
+# import numpy as np
+import scanpy as sc
+
+
+def _dca(adata):
+    from dca.api import dca
+
+    # sc.AnnData takes (counts, obs=[obs], vars=[vars]), but returns an anndata
+    # even if just given counts.
+    adata2 = sc.AnnData(adata.obsm["train"])
+    dca(adata2, threads=1)
```

Comment:
You will want to set test args (e.g. `epochs`, see https://github.com/theislab/dca/blob/master/dca/api.py#L31). Example: https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/label_projection/methods/mlp.py#L10

Reply:
Example commit: https://github.com/giovp/SingleCellOpenProblems/commit/58fab2af37d56729d38ed3e8df73ffcbaed63122